frontend/src/modules/entitydetails/components/Screenshots.test.js

```diff
@@ -1,23 +1,20 @@
-import React from 'react';
-import { shallow } from 'enzyme';
-import sinon from 'sinon';
+import * as lightbox from 'core/lightbox';
 
 import Screenshots from './Screenshots';
+import { createReduxStore, mountComponentWithStore } from 'test/store';
 
 describe('<Screenshots>', () => {
     it('shows a Lightbox on image click', () => {
+        const store = createReduxStore();
         const source = 'That is an image URL: http://link.to/image.png';
-        const openLightboxFake = sinon.fake();
-        const wrapper = shallow(
-            <Screenshots
-                source={source}
-                locale='kg'
-                openLightbox={openLightboxFake}
-            />,
-        );
-
+        const wrapper = mountComponentWithStore(Screenshots, store, {
+            locale: 'kg',
+            source,
+        });
         wrapper.find('img').simulate('click');
 
-        expect(openLightboxFake.calledOnce).toEqual(true);
+        const newState = store.getState()[lightbox.NAME];
```

Comment:
I'm not too fond of the test here.

In a perfect world, I'd love the test here to pair with the test in https://github.com/mozilla/pontoon/blob/f000a47d7ff18cdae1c84b306711d34816f1db52/frontend/src/core/lightbox/components/Lightbox.test.js#L34-L45, i.e., catch the call to `dispatch`.

Mocking `store.dispatch` didn't seem to do that, much to my surprise.

Reply:
Figured it out, had to mock before mounting.